echo_handlers.go

```diff
@@ -224,6 +229,8 @@ func (svc *Service) AppsCreateHandler(c echo.Context) error {
 	app := App{Name: name, NostrPubkey: pairingPublicKey}
 	maxAmount, _ := strconv.Atoi(c.FormValue("MaxAmount"))
 	maxAmountPerTransaction, _ := strconv.Atoi(c.FormValue("MaxAmountPerTransaction"))
+	budgetType := c.FormValue("BudgetType")
+	expiresAt, _ := time.Parse("2021-01-21", c.FormValue("ExpiresAt"))
```

Comment:
expiresAt seems to be 00:00:00+00:00 even if I select a valid date. It is passed in the form correctly. 

Maybe you shouldn't ignore the error?

Also, in the datepicker I can select today's date. But I guess that would be the start of the day, which would mean the app is already expired? Maybe we should use the end of the selected day?